src/ssl.c

```diff
@@ -23670,11 +23670,45 @@ long wolfSSL_set_tlsext_debug_arg(WOLFSSL* ssl, void *arg)
 #endif /* HAVE_PK_CALLBACKS */
 
 #if defined(OPENSSL_ALL) || defined(WOLFSSL_HAPROXY)
-const unsigned char *SSL_SESSION_get0_id_context(const WOLFSSL_SESSION *sess, unsigned int *sid_ctx_length)
+const unsigned char *SSL_SESSION_get0_id_context(const WOLFSSL_SESSION *sess,
+                                                 unsigned int *sid_ctx_length)
 {
     sess = ClientSessionToSession(sess);
     return wolfSSL_SESSION_get_id((WOLFSSL_SESSION *)sess, sid_ctx_length);
 }
+WOLFSSL_API int SSL_SESSION_set1_id(WOLFSSL_SESSION *s,
```

Comment:
Please us our normal mapping for the functions. Use `wolfSSL_SESSION_set1_id` here. Then add a `#define SSL_SESSION_set1_id wolfSSL_SESSION_set1_id` in the openssl/ssl.h. Please do this for the other functions here as well. I think these previously stubbed functions took shortcuts. We do this mapping to avoid function name collisions.

Reply:
The `WOLFSSL_API` is only needed in the header.